.github/workflows/prepare-release-branch.yml

```diff
@@ -0,0 +1,100 @@
+name: Prepare release branch
+on:
+  workflow_dispatch:
+
+jobs:
+  create-pull-request-against-release-branch:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Create release branch
+        id: create-release-branch
+        run: |
+          stable_version=$(./scripts/eachdist.py version --mode stable)
+          stable_version=${stable_version//-dev/}
+
+          prerelease_version=$(./scripts/eachdist.py version --mode prerelease)
+          prerelease_version=${prerelease_version//-dev/}
+
+          stable_version_branch_part=$(echo $stable_version | sed -E 's/([0-9]+)\.([0-9]+)\.0/\1.\2.x/')
+          prerelease_version_branch_part=$(echo $prerelease_version | sed -E 's/0\.([0-9]+)b0/0.\1bx/')
+          release_branch_name="release/v$stable_version_branch_part-$prerelease_version_branch_part"
+
+          git push origin HEAD:$release_branch_name
+
+          echo "STABLE_VERSION=$stable_version" >> $GITHUB_ENV
+          echo "PRERELEASE_VERSION=$prerelease_version" >> $GITHUB_ENV
+          echo "RELEASE_BRANCH_NAME=$release_branch_name" >> $GITHUB_ENV
+
+      - name: Update version
+        run: .github/scripts/update-version.sh $STABLE_VERSION $PRERELEASE_VERSION
+
+      - name: Set git user
+        run: .github/scripts/set-git-user.sh
```

Comment:
Curious, what is the purpose for setting the user? Is this for permissions to push to origin branches?

Reply:
This is needed so that the PRs will pass the EasyCLA check, see https://github.com/open-telemetry/community/issues/809#issuecomment-1015121722